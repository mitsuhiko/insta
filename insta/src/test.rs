#[test]
fn test_embedded_test() {
    assert_snapshot!("embedded", "Just a string");
}
