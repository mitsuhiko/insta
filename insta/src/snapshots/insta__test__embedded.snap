---
source: insta/src/test.rs
expression: "\"Just a string\""
---
Just a string
